core/src/ldml/ldml_transforms.hpp

```diff
@@ -270,6 +271,17 @@ class transforms {
        const kbp::kmx::COMP_KMXPLUS_TRAN_Helper &tranHelper);
 };
 
+// string routines
+
+/** Normalize a u32string inplace. Returns a reference to the same string. */
+std::u32string &normalize_nfd(std::u32string &str, UErrorCode &status);
+/** Normalize a u16string inplace. Returns a reference to the same string. */
+std::u16string &normalize_nfd(std::u16string &str, UErrorCode &status);
+/** Normalize a u32string inplace. Returns a reference to the same string. */
+std::u32string &normalize_nfc(std::u32string &str, UErrorCode &status);
+/** Normalize a u16string inplace. Returns a reference to the same string. */
+std::u16string &normalize_nfc(std::u16string &str, UErrorCode &status);
```

Comment:
The comment should say what the normalization is otherwise it has less information than the function name!

```suggestion
/** Normalize a u32string inplace to NFD. Returns a reference to the same string. */
std::u32string &normalize_nfd(std::u32string &str, UErrorCode &status);
/** Normalize a u16string inplace to NFD. Returns a reference to the same string. */
std::u16string &normalize_nfd(std::u16string &str, UErrorCode &status);
/** Normalize a u32string inplace to NFC. Returns a reference to the same string. */
std::u32string &normalize_nfc(std::u32string &str, UErrorCode &status);
/** Normalize a u16string inplace to NFC. Returns a reference to the same string. */
std::u16string &normalize_nfc(std::u16string &str, UErrorCode &status);
```

Reply:
originally it was all nfd!